railties/CHANGELOG.md

```diff
@@ -5,12 +5,19 @@
 
     *Richard Schneeman*
 
-*   Support environment specific credentials file.
+*   Support environment specific credentials overrides.
 
-    For `production` environment look first for `config/credentials/production.yml.enc` file that can be decrypted by
-    `ENV["RAILS_MASTER_KEY"]` or `config/credentials/production.key` master key.
-    Edit given environment credentials file by command `rails credentials:edit --environment production`.
-    Default paths can be overwritten by setting `config.credentials.content_path` and `config.credentials.key_path`.
+    So any environment will look for `config/credentials/#{Rails.env}.yml.enc` and fall back
+    to `config/credentials.yml.enc`.
+
+    The encryption key can be in `ENV["RAILS_MASTER_KEY"]`, `config/credentials/production.key` or `config/credentials/master.key`.
```

Comment:
Would `config/credentials/development.yml.enc` and `config/credentials.yml.enc` share the same master key? It seems like this is answered in the `USAGE` docs below but I'm honestly still not 100% sure. Could def use some clarification, and certainly could use highlighting of whether it's the same key vs different keys in the changelog IMO.

Reply:
Good point! We actually don't fallback to the `master.key` file, which I think I thought back when doing this editing.